cmd/cli/deprecated/exec.go

```diff
@@ -0,0 +1,17 @@
+package deprecated
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewExecCommand() *cobra.Command {
+	cancelCmd := &cobra.Command{
+		Use:        "exec",
+		Deprecated: "exec was an experimental feature and no longer supported",
+		RunE: func(cmd *cobra.Command, cmdArgs []string) error {
+			return nil
```

Comment:
might return an error here too

Reply:
not necessary and follows the same pattern as with other deprecated commands:

```
[bacalhau]→ bacalhau exec                                                                                        
Command "exec" is deprecated, exec was an experimental feature and no longer supported

```